src/test/java/com/brandwatch/internship/fetchandstoretweets/services/QueryServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@TestPropertySource(locations="classpath:test.properties")
+public class QueryServiceTest {
+
+    private static final String EMPTY_SEARCH_STRING = "";
+
+    @Mock
+    private QueryRepository queryRepository;
+
+    @InjectMocks
+    private QueryService queryService;
+
+    @Before
+    public void mockQueryRepository() {
+        when(queryRepository.findOneById(anyLong())).thenReturn(null);
+    }
+
+    @Test(expected = Query.EmptyQuerySearchStringException.class)
+    public void createQueryExceptionTest() {
+        queryService.createQuery(EMPTY_SEARCH_STRING);
+    }
+
+    @Test(expected = Query.NoSuchQueryException.class)
+    public void getQueryByIdExceptionTest() {
+        queryService.getQueryById(anyLong());
```

Comment:
I think `anyLong()` method should be used in mock definition: 
```
  when(queryRepository.findOneById(anyLong())).thenReturn(null);
```
but in method calls you should use constant values, e.g.
```
private static final long QUERY_ID = 137;
...
queryService.getQueryById(QUERY_ID);
```

Reply:
I thought this too! so I wanted to see if that is the case or not. I thought that it would throw an error but it did not. Because of that I left it saying that this is more general. When you told me to remove the other annotations and just use mockito, I got the error I was expecting!!